recipes/samtools/0.1.12/meta.yaml

```diff
@@ -24,4 +24,8 @@ test:
   files:
     - ex1.fa
   commands:
-    - "samtools faidx ex1.fa"
+    - samtools faidx ex1.fa
+    - "maq2sam-short 2>&1 | grep '^Version: r439'"
+    - "maq2sam-long 2>&1 | grep '^Version: r439'"
+    - "wgsim 2>&1 | grep '^Version: 0.3.2'"
```

Comment:
typo in `wgsim` version: `0.2.3`
Also it might be best to avoid single quotes in the strings' contents for the moments, use instead:
```yaml
    - 'maq2sam-short 2>&1 | grep "^Version: r439"'
    - 'maq2sam-long 2>&1 | grep "^Version: r439"'
    - 'wgsim 2>&1 | grep "^Version: 0\.2\.3"'
```